src/components/moleculs/post/Post.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import s from "./Post.module.css";
+import cn from "classnames";
+import img from "../../../assets/postImg.png";
+
+interface IPosts {
+	myPost?: boolean;
```

Comment:
1) IPosts -> IPost
2)`interface  IPost {
img: ..,
title: ...,
.....,
name?: string
}`

 better remove this field 
myPost?: boolean;

and field s.name make optional